working/base-interface-final/feature-specification.md

```diff
@@ -0,0 +1,470 @@
+# "Base", "interface", and "final" types
+
+Author: Bob Nystrom
+
+Status: In-progress
+
+Version 1.0
+
+This proposal specifies three modifiers that can be placed classes and mixins to
+allow an author to control whether the type allows being implemented, extended,
+both, or neither. The proposed modifiers are:
+
+*   No modifier: As today, the type has no restrictions.
+*   `base`: The type can be extended or mixed in but not implemented.
```

Comment:
Could we remove the "mixed in" and require you to write `mixin` to get that.

Then `mixin` could combine with `base class` or plain `class`, to allow the class (with only `Object` as superclass and no declared constructors) to be used as a mixin.
Then we can finally stop assuming that *any* class satisfying those random requirements can potentially be mixed in.

Reply:
Done.